app/controllers/course/user_invitations_controller.rb

```diff
@@ -164,7 +164,7 @@ def aggregate_errors
   # @return [Array<String>]
   def invalid_course_user_errors
     invalid_course_users.map do |course_user|
-      user = self.class.helpers.display_user(course_user.user || course_user)
+      user = self.class.helpers.display_course_user(course_user.user || course_user)
```

Comment:
Do all courses have a name if invalid ?  I think we can use `course_user` as param if that's the case, also not sure why user is used before.